src/components/ui/Alert.jsx

```diff
@@ -28,6 +29,11 @@ const Alert = ({
     <div className="alert-content">
       <span role="alert">{description}</span>
     </div>
+    <div className="alert-link">
+      <a onClick={onClose}>
+        { _('Do not display this message again') }
```

Comment:
If we want to re-use the Alert component, this shouldn't be hardcoded but passed as optional prop.